store/reducers/aliasData.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file aliasCache.ts
+ *
+ * Reducers for handling data by URL path alias.
+ */
+
+import { AnyAction } from 'redux';
+import { HYDRATE } from 'next-redux-wrapper';
+import { ContentDataState, RootState } from '@interfaces/state';
+import { generateLinkHrefForContent } from '@lib/routing/content';
+
+type State = ContentDataState | RootState;
+
+export const aliasData = (state: State = {}, action: AnyAction) => {
```

Comment:
They app rewrites all unknown routes to `/`, passing the pathname as an `alias` param. That alias is then sent to the Drupal API to get basic resource info for the requested content, such as resource type and id, which is used to determine what page component is used to render the page. The page component is then responsible for using the id to request a complete set of data for the page. This store is so that initial basic info request can be skipped if we had already collected that info in another way, ie. expanded reference data in full content request, or requests for lists of content like a landing page or latest stories list.